### DISCUSSION TITLE
HttpTransferManager Uploading without Multi-part form data

### DISCUSSION THREAD

**Discussion by bb281** [shinyorg/shiny#1455] [Q&A]
My goal is to upload multiple photos to Azure Blob Storage with the HttpTransferManager.  

I was previously writing the raw blob to Blob storage using HttpClient. After swapping to the HttpTransferManager, I noticed the body is not the raw blob/photo and has the multipart/form data/boundary.

Is there a way to use HttpTransferManager without multipart/form data so the data is just the raw blob?

**Comment by aritchie** [comment]
That is not supported at this time
